app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AdminsController < ApplicationController
+  def approve_user
```

Comment:
better to rename it to `approved_users`

Reply:
Refactored and removed this block of code